pkg/pillar/cmd/diag/diag.go

```diff
@@ -68,55 +70,71 @@ type diagContext struct {
 }
 
 // Set from Makefile
-var Version = "No version specified"
+var (
```

Comment:
Can we put these in a diagContext struct instead of being free standing variables?